packages/service-library/src/servicelib/aiohttp/openapi_wrappers.py

```diff
@@ -113,7 +115,7 @@ def __init__(self, response: web.Response, data: str):
 
     @staticmethod
     async def create(response: web.Response):
-        text = await response.text()
+        text = await response.text
```

Comment:
are you sure about this one? I think aiohttp has a function text(). Would you mind double checking it?

Reply:
https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.Response.text